server/mscalendar/daily_summary.go

```diff
@@ -127,7 +127,7 @@ func (m *mscalendar) ProcessAllDailySummary(now time.Time) error {
 
 		shouldPost, shouldPostErr := shouldPostDailySummary(dsum, now)
 		if shouldPostErr != nil {
-			m.Logger.Warnf("Error posting daily summary for user %s. err=%v", storeUser.MattermostUserID, shouldPostErr)
+			m.Logger.With(bot.LogContext{"mm_user_id": storeUser.MattermostUserID, "now": now.String()}).Warnf("Error checking daily summary should be posted")
```

Comment:
We're no longer logging `shouldPostError`. Is this intentional?

Reply:
I missed that.